frontend/src/queries/schema.ts

```diff
@@ -1979,3 +1983,31 @@ export interface TrendsAlertConfig {
 export interface HogCompileResponse {
     bytecode: any[]
 }
+
+export interface TeamTaxonomyResponse {
```

Comment:
`...Response` naming only applies to the final whole response, while this is a subpart of it. Something more specific would be clearer here, e.g. `EventInfo`